### ISSUE TITLE
valid-title is not triggered for disallowedWord with special character

### ISSUE THREAD

**Issue by aaron-goff** [playwright-community/eslint-plugin-playwright#341]
When trying to modify our usage of `valid-title`, I wanted to add a disallowedWord for `'@new'`, as we somewhat frequently use this for ease of use in finding just the new tests we're working on writing. With the following setup in our `.eslintrc.cjs` file:

```
...
'playwright/valid-title': [
          'error',
          {
            disallowedWords: ['@new']
          }
        ]
...
```

The following is still considered valid:

```javascript
test('@new', async () => {
    expect(true).toBe(true);
  });
```

I've tested this with `!`, `@`, `#`, and `$`, and none are picked up by the linter. I also validated the rest of my setup was fine, by changing `'@new'` to `'new'`, which does trigger the linting error.

**Comment by mskelton** [comment]
Going to close in favor of #342, not sure I want to mess with this in valid-title since using tags in titles isn't really the recommended approach anymore with the `tags` option.
